src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -42,7 +42,8 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Taint
     [TestClass]
     public class TaintIssuesSynchronizerTests
     {
-        private const string SharedProjectKey = "test";
```

Comment:
Lots of the tests were using this constant in an opaque way e.g. making an implicit assumption that some of the services had been set up with this key. Some checks were using this key when they didn't need to.

All in all, it was very difficult to tell by reading a test what data it actually required to pass.

I've removed this constant and made all of the tests specify the data they need explicitly.